### PR TITLE
Adding ISSUE_TEMPLATE and new pull_request  Template 

### PR DESCRIPTION
**Pull request by zeel-codder**
zeel hear,  In this pull request, I  add the two custom new ISSUE TEMPLATE  and also the new theme add pull request.

Thank you

### CODE REVIEW

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Thank you for the PR @zeel-codder!

Could please also add another issue template for other issues? Not all issues we had in the past fall under the _bug_ and _docs_ categories. I'm thinking of something simple like this:

```md
---
name: Other
about: An issue or question that doesn't fit in the other categories
---

```

(Feel free to add a fitting emoji.)

Regarding the PR template: We have more than just PRs that add new themes. I think a template as simple as the following might be enough.

```md
<!--
Before submitting your PR, please make sure that all tests pass. 
You can run all tests using `npm test`.
-->

```

**Comment by zeel-codder** [comment]
I make the changes As per your advice.
Thank you

File: .github/ISSUE_TEMPLATE/blug.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
[the remark this answers is not in the document]
Comment (RunDevelopment) [reply .github/ISSUE_TEMPLATE/blug.md:1]: Could you please fix the typo in the file name?

File: .github/ISSUE_TEMPLATE/other.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: Other
+about: An issue or question that doesn't fit in the other categories
+---
+
+## 🤖 Other <!-- Enter issue name or some main key point about issus -->
+
+<!-- A clear and concise description of what the issue is. -->
```
Comment (RunDevelopment) [inline .github/ISSUE_TEMPLATE/other.md:8]: I would like it to be a little simpler.

```suggestion
<!-- A clear and concise description of what the issue is. -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## 🙌 Thanks For Making PR
+
+<!--
```
Comment (RunDevelopment) [inline .github/pull_request_template.md:3]: People will not remove this heading. I don't want most the PRs we get start with "🙌 Thanks For Making PR", so let's move this into the comment.
```suggestion
<!--
🙌 Thank you for making this PR!

```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Example how to mark a checkbox :-
+- [x] My code follows the code style of this project.
+-->
+- [ ] My code follows the code style of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] My changes generate no new warnings.
+- [ ] I have run the test cases.
+- [ ] This PR is for Adding New Theme
+
+
```
Comment (RunDevelopment) [inline .github/pull_request_template.md:19]: I think the checklist is a little too long. Also, I think it should be a comment again. A lot of people just leave these types of checklists as is.

I don't think we need to encourage people to describe their work. The titles and descriptions people choose are typically enough.

```suggestion
Before submitting your PR, please check the following:

- Your code follows the code style of this project (`npm run lint`).
- You have performed a self-review of your own code.
- You have run the test cases (`npm run test`).
-->
```

**Review by RunDevelopment — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 4 above)
Comment (zeel-codder) [reply .github/pull_request_template.md:19]: hmm, I will change this.

**Comment by zeel-codder** [comment]
I make the changes. if you want me to add more issue templates or anything else feel free to tell me, I will happy to do that.
thank you

**Review by RunDevelopment — APPROVED** [review]

**Comment by RunDevelopment** [comment]
Thank you very much for contributing @zeel-codder!

**Pull request merged**